abr_control/controllers/osc.py

```diff
@@ -98,21 +135,24 @@ def generate(self, q, dq,
 
         # calculate the inertia matrix in joint space
         M = self.robot_config.M(q)
+        self.M = M
 
         # calculate the inertia matrix in task space
         M_inv = np.linalg.inv(M)
+        self.M_inv = M_inv
         # calculate the Jacobian for end-effector with no offset
         Mx_inv = np.dot(J, np.dot(M_inv, J.T))
-        if np.linalg.det(Mx_inv) != 0:
+        if np.linalg.det(Mx_inv) > 1e-4:
```

Comment:
I don't remember if we wanted this to be zero or not

Reply:
this has been updated (and corrected) in master